test/ecto/query/builder/filter_test.exs

```diff
@@ -4,6 +4,14 @@ defmodule Ecto.Query.Builder.FilterTest do
   import Ecto.Query.Builder.Filter
   doctest Ecto.Query.Builder.Filter
 
+  def strip_heredoc(string) do
```

Comment:
Ahh but I had so much fun figuring that out :relaxed: . Sorry I will get out of the bike shed.